src/rpc/rpcquorums.cpp

```diff
@@ -360,7 +363,17 @@ UniValue quorum_sigs_cmd(const JSONRPCRequest& request)
     uint256 msgHash = ParseHashV(request.params[3], "msgHash");
 
     if (cmd == "sign") {
-        return llmq::quorumSigningManager->AsyncSignIfMember(llmqType, id, msgHash);
+        if(request.params.size() > 4){
```

Comment:
Nit: I think it's clear and my personal preference is to see formatting as defined in the `.clang-format` file for new code, e.g. spaces before and after the outer parens.